easybuild/easyconfigs/m/MDSplus/MDSplus-7.132.0-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,77 @@
+easyblock = 'ConfigureMake'
+
+name = 'MDSplus'
+version = '7.132.0'
+
+homepage = 'http://mdsplus.org/'
+description = """MDSplus is a set of software tools for data acquisition
+  and storage and a methodology for management of complex scientific data."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+toolchainopts = {'pic': True}
+
+sources = [{
+    'filename': SOURCELOWER_TAR_GZ,
+    'git_config': {
+        'url': 'https://github.com/' + name,
+        'repo_name': '%(namelower)s',
+        'commit': 'stable_release-' + version.replace('.', '-'),
+        'recursive': True,
+    },
+}]
+patches = ['%(namelower)s-%(version)s_paths.patch']
+checksums = [
+    {'mdsplus-7.132.0.tar.gz': '130b584c426e09167e042f2da829c03f59c57e9b352844bb4ef0880ccf93c8a7'},
```

Comment:
Since this is pulled down using git_config it will not have a stable checksum.
I.e. you need to change it to
```
{'mdsplus-7.132.0.tar.gz': None},
```
But I'm not sure if we've manged to fix the bugs we had in framework regarding this.

It would be better to pull down the https://github.com/MDSplus/mdsplus/archive/refs/tags/stable_release-7-132-0.tar.gz file and the corresponding version of https://github.com/MDSplus/3rd-party-apis.git unfortunately it doesn't have releases, so the problem will remain...

Reply:
Thanks @akesandgren, this is good to know!  I've updated the EB file to reflect this.  I had to change the syntax slightly since `--check-style` wasn't happy with what you proposed.